### ISSUE TITLE
Since #11786 the label receives focus before the url

### ISSUE THREAD

**Issue by TheOneRing** [owncloud/client#11800]
- The label receives focus
- the focus is not visible

**Comment by erikjv** [comment]
Yes, and on the next tab, the label is focused. It is handled inside the QLabel in a weird way: if you back-tab into the label, the link is not selected, and you then have to tab (NOT back-tab) to select it.

**Comment by TheOneRing** [comment]
We could use two labels with a buddy?
One with just the url and we directly handle the keyboard interaction?

**Comment by erikjv** [comment]
We decided to remove the URL completely, and only show the connection status. We also added a "Open in browser" entry in the "Manage Account" button.

**Comment by saw-jan** [comment]
Tested on `ownCloud 6.0.0.14688-daily20240918 [967f29]`

![Screenshot from 2024-09-18 18-14-24](https://github.com/user-attachments/assets/97f202b1-9420-4b2b-96f5-220ef9d36983)
